pkg/policycache/informer.go

```diff
@@ -67,8 +68,12 @@ func (c *Controller) updatePolicy(old, cur interface{}) {
 }
 
 func (c *Controller) deletePolicy(obj interface{}) {
-	p := obj.(*kyverno.ClusterPolicy)
-	c.Cache.remove(p)
+	p, ok := kubeutils.GetObjectWithTombstone(obj).(*kyverno.ClusterPolicy)
+	if ok {
+		c.Cache.remove(p)
+	} else {
+		logger.Info("Failed to get deleted object", "obj", obj)
```

Comment:
What does it mean if it fails to get deleted policy? Do we not want to remove it from the policy cache?

Reply:
We can't remove it from the policy cache as we don't have the object...

Reply:
Can we be more specific on the error message? Saying that the policy cannot be removed from the cache.

Reply:
Good idea, I will update the message.

Reply:
Done.